sparse_ho/algo/implicit.py

```diff
@@ -39,20 +37,21 @@ def get_beta_jac_v(
 
 def get_beta_jac_t_v_implicit(
```

Comment:
nitpick: it's beta_jac_v in other parts of the code, right ?
also, there's no docstring

Reply:
I changed:  ` get_beta_jac_v` >> `compute_beta_grad`